cpp/memilio/utils/date.h

```diff
@@ -327,4 +360,4 @@ inline int get_offset_in_days(Date date1, Date date2)
 
 } // end namespace mio
 
-#endif // EPI_UTILS_DATE_H
+#endif // EPI_UTILS_DATE_H
```

Comment:
Please add the newline back. You can tick the "Insert Final Newline" option in vscode to do that automatically.